### PR TITLE
Fix some comment mismatch.

### PR DESCRIPTION
**Pull request by wanguanglu**

### CODE REVIEW

File: src/hmm/transition-model.h [hunk 1]
```diff
@@ -288,8 +288,8 @@ class TransitionModel {
 
   HmmTopology topo_;
 
-  /// Triples indexed by transition state minus one;
-  /// the triples are in sorted order which allows us to do the reverse mapping from
+  /// Tuples indexed by transition state minus one;
+  /// the tuples are in sorted order which allows us to do the reverse mapping from
   /// triple to transition state
```
Comment (danpovey) [inline src/hmm/transition-model.h]: there is one  more 'triple' on this line.

Comment (wanguanglu) [reply src/hmm/transition-model.h]: yeal, I fixed it.

**Comment by wanguanglu** [comment]
I don't know why the continuous integration sometimes fail?

**Comment by danpovey** [comment]
the travis build sometimes fails due to issues in how it was set up.  i am hoping that e.g. @galv might have a chance to figure out what's going wrong-- I haven't had the time.

**Pull request merged**